lesson3.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('js-list');
+for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    const img = document.createElement('img');
+    const imgbox = ['/img/bookmark.png','/img/message.png'];
```

Comment:
MUST
imgbox does not need to be turned in for. Please define and refer to it outside of for

Reply:
I learned that imgbox does not need to be repeated. Thank you for pointing this out.